module/category/src/Infrastructure/Persistence/Query/DbalTreeQuery.php

```diff
@@ -70,6 +70,35 @@ public function findTreeIdByCode(string $code): ?CategoryTreeId
         return null;
     }
 
+    public function autocomplete(
+        Language $language,
+        string $search = null,
+        int $limit = null,
+        string $field = null,
+        ?string $order = 'ASC'
+    ): array {
+        $query = $this->connection->createQueryBuilder()
+            ->select('id, code, COALESCE(name->>:language, null) as label')
```

Comment:
COALESCE is not needed probably?
```suggestion
            ->select('id, code, name->>:language as label')
```

Reply:
I think it is need as name is not required  and can be empty an array.

Reply:
Basically what you're doing with COALESCE is saying if the value is null return null... So I think it's not ;)